pkgs/development/python-modules/reportlab/default.nix

```diff
@@ -25,6 +25,9 @@ in buildPythonPackage rec {
   postPatch = ''
     rm tests/test_graphics_barcode.py
     rm tests/test_graphics_render.py
+    rm tests/test_lib_utils.py
+    rm tests/test_platypus_breaking.py
+    rm tests/test_platypus_general.py
```

Comment:
You should add comments explaining why which test is removed.

Reply:
done